processor/src/main/java/com/ihren/processor/config/ObjectMapperConfig.java

```diff
@@ -7,7 +7,8 @@
 public class ObjectMapperConfig {
     public static void configure(ObjectMapper objectMapper) {
         objectMapper
+                .findAndRegisterModules()
```

Comment:
All necessary modules should be added explicitly not to inject something unexpectedly